### PR TITLE
Created deletion page UI and fixed issues from the review

### LINKED ISSUE #4: Create note deletion page UI

### PR DESCRIPTION
**Pull request by gaagul**
Fixes #4

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx [hunk 1]
```diff
@@ -1,42 +1,14 @@
...
-      onClose={onClose}
-      onSubmit={handleDelete}
-    />
-  );
-};
-
+const DeleteAlert = ({ onClose, selectedNote, handleDelete }) => (
+  <Alert
+    isOpen
+    message="Are you sure you want to continue? This cannot be undone."
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx:8]: Go through the Slack thread.

File: app/javascript/src/components/Dashboard/Notes/NoteCard.jsx [hunk 2]
```diff
@@ -3,35 +3,40 @@ import React from "react";
...
         <li>Edit</li>
-        <li onClick={setShowDeleteAlert}>Delete</li>
+        <li
+          onClick={() => {
+            setShowDeleteAlert(true);
+            setSelectedNote(note);
+          }}
+        >
+          Delete
+        </li>
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/NoteCard.jsx:21]: https://github.com/gaagul/neeto-ui-challenge/pull/9#discussion_r978384451

File: app/javascript/src/components/Dashboard/Notes/NoteCard.jsx [hunk 3]
```diff
@@ -3,35 +3,40 @@ import React from "react";
...
+    <Typography style="body2" weight="light">
+      {note.description}
+    </Typography>
     <hr className="my-2" />
     <div className="flex justify-between">
       <Tag label="Getting Started" />
       <div className="flex items-center">
         <Clock size={14} />
-        <Tooltip content={dayjs().format("dddd, hh:mmA")} position="bottom">
+        <Tooltip content={formatTime(note.createdAt)} position="bottom">
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/NoteCard.jsx:32]: `formatTime` is a vague util name. Format time into what? Find a better name from business perspective.

File: app/javascript/src/components/Dashboard/Notes/Pane/Create.jsx [hunk 4]
```diff
@@ -6,7 +6,7 @@ import Form from "./Form";
 
 import { NOTES_FORM_INITIAL_FORM_VALUES } from "../constants";
 
-const NewNotePane = ({ fetchNotes, showPane, setShowPane }) => {
+const NewNotePane = ({ showPane, setShowPane }) => {
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Pane/Create.jsx:9]: What is the advantage of NOT naming this component as `Create` itself which is filename?

File: app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx [hunk 5]
```diff
@@ -8,7 +8,7 @@ import notesApi from "apis/notes";
 
 import { NOTES_FORM_VALIDATION_SCHEMA } from "../constants";
 
-const NoteForm = ({ onClose, refetch, note, isEdit }) => {
+const NoteForm = ({ onClose, note, isEdit }) => {
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx:11]: Same as last question.

File: app/javascript/src/components/Dashboard/Notes/constants.js [hunk 6]
```diff
@@ -9,3 +9,35 @@ export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
   title: yup.string().required("Title is required"),
   description: yup.string().required("Description is required"),
 });
+
+export const NOTES_CARD_DATA = [
+  {
+    id: 1,
+    title: "How to claim the warranty?",
+    description:
+      '"Are you getting my texts???" she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn\'t getting',
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/constants.js:18]: Why are you enclosing this string within single quotes? Go through the string content and verify the text and the necessity of quotes within the text. Here you should be using double quotes to enclose the whole string.

**Review by yedhink — COMMENTED** [review]
@gaagul

File: app/javascript/src/components/Dashboard/Notes/NoteCard.jsx (hunk 3 above)
Comment (gaagul) [reply app/javascript/src/components/Dashboard/Notes/NoteCard.jsx:32]: I have changed the util name to formatToDayTime.

File: app/javascript/src/components/Dashboard/Notes/Pane/Create.jsx (hunk 4 above)
Comment (gaagul) [reply app/javascript/src/components/Dashboard/Notes/Pane/Create.jsx:9]: This was already present when I forked from the wheel, I thought that was the concept we followed and did not touch that part
wheel-Create jsx at master · bigbinary-wheel 2022-09-23 15-46-21

File: app/javascript/src/components/Dashboard/Notes/constants.js (hunk 6 above)
Comment (gaagul) [reply app/javascript/src/components/Dashboard/Notes/constants.js:18]: There were double quotes already in the note title in the Figma design which is why I passed the whole string in single quotes.
I tried using the escape character but prettier was automatically changing it to single quotes.

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Notes/constants.js:18]: No need for the double quotes within the string content.

File: app/javascript/src/components/Dashboard/Notes/Pane/Create.jsx (hunk 4 above)
Comment (gaagul) [reply app/javascript/src/components/Dashboard/Notes/Pane/Create.jsx:9]: I will make sure this will not happen in future.

**Pull request merged**